src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+import java.util.Arrays;
 public class Duke {
+
+    private static Scanner in = new Scanner(System.in);
+    private static final Task[] tasks = new Task[100];
+
+    public static void command(String line) {
+        String exitCommand = "bye";
+        String toDo = "list";
+        String checkAsDone = "mark";
+        int numberOfTasks = 0;
+
+        String[] tokens = line.split(" ", 2);
+        String command = tokens[0];
+
+        while (!command.equals(exitCommand)) {
+            if (tokens[0].equals(toDo)) {
+                System.out.println("Here are your list of tasks to complete!");
+                for (int i = 0; i < numberOfTasks; ++i) {
+                    System.out.format("%d. ", (i + 1));
+                    System.out.println(tasks[i]);
+                }
+                line = in.nextLine();
+                tokens = line.split(" ", 2);
+                command = tokens[0];
+            } else if (command.equals(checkAsDone)){
+                int finishedTask = Integer.parseInt(tokens[1]) - 1;
+                tasks[finishedTask].markAsDone();
+                System.out.println("Alright! I have marked the task as complete!");
+
+                for (int i = 0; i < numberOfTasks; ++i) {
+                    System.out.format("%d. ", (i + 1));
+                    System.out.println(tasks[i]);
+                }
+
+                line = in.nextLine();
+                tokens = line.split(" ", 0);
+                command = tokens[0];
+            } else {
+                String task = tokens[1];
+                switch(command) {
+                case "todo":
+                    Task t = new ToDo(task);
```

Comment:
We want to avoid using a single character as the name of a variable (with some exceptions such as i, j, k in a for loop). Try to think if another name for this